playbooks/roles/shadowsocks/templates/instructions.md.j2

```diff
@@ -64,6 +64,17 @@ For reference, this is the configuration specified:
         Password: {{ shadowsocks_password.stdout }}
         Encryption Method: {{ shadowsocks_encryption_method }}
 
+For users on unreliable or hostile networks (esp. experiencing quality-of-service (QOS) throttling), further configuration of Shadowsocks to use the simple-obfs plugin can be carried out via the following configuration:
+
+        Server: {{ streisand_ipv4_address }}
+        Port: {{ shadowsocks_server_port }}
+        Password: {{ shadowsocks_password.stdout }}
+        Encryption Method: {{ shadowsocks_encryption_method }}
+        Plugin: {{ shadowsocks_client_obfs_plugin }}
+        Plugin_Options: {{ shadowsocks_obfs_client_plugin_opts }}
+
+Your Shadowsocks traffic will now be obfuscated as HTTP traffic from `{{ shadowsocks_obfs_cover_domain }}`.
```

Comment:
I think this should actually read "will now be obfuscated as HTTP traffic **to** `{{ shadowsocks_obfs_cover_domain }}`" no?